tools/sigh.ts

```diff
@@ -1084,6 +1079,17 @@ async function runSteps(command: string, args: string[]): Promise<boolean> {
   globalOptions = {...globalOptions, ...minimist(args, {
     boolean: ['install'],
   })};
+  for (const key in globalOptions) {
```

Comment:
I am not sure what this block is doing, are you just converting string values to boolean?
add a comment to explain what you are doing and why